parser_library/src/diagnostic.cpp

```diff
@@ -2059,6 +2059,22 @@ diagnostic_op diagnostic_op::error_E073(const range& range)
         diagnostic_severity::error, "E073", "Illegal START instruction - CSECT already exists.", range);
 }
 
+diagnostic_op diagnostic_op::error_E074(const range& range)
+{
+    return diagnostic_op(diagnostic_severity::error, "E074", "Generated operation code is null.", range);
+}
+
+diagnostic_op diagnostic_op::error_E075(std::string_view message, const range& range)
+{
+    return diagnostic_op(diagnostic_severity::error,
+        "E075",
+        concat("The name field contains a non-alphanumeric character, that is, a character other than A to Z, 0 to 9, "
+               "$, #, "
+               "@, or _. - ",
```

Comment:
$ or # are not alphanumeric either, so maybe just invalid characters?

Also the end of the sentence looks a bit weird - `... @, or _. - <more text>`

Reply:
👍